src/editor/Editor.js

```diff
@@ -98,6 +98,7 @@ define(function (require, exports, module) {
         WORD_WRAP           = "wordWrap",
         INDENT_LINE_COMMENT = "indentLineComment",
         ALLOW_JAVASCRIPT    = "allowJavaScript",
+        ALLOW_WHITESPACE    = "allowWhiteSpace",
```

Comment:
Same with this file, none of these changes are necessary.  Please revert them with:

```
git checkout master src/editor/Editor.js
```